recipes/cuda-cupti/meta.yaml

```diff
@@ -0,0 +1,173 @@
+{% set name = "cuda-cupti" %}
+{% set version = "12.0.90" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cupti/{{ platform }}/cuda_cupti-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 3c93a6a8c5eab3a56c1a0a514c0ac7d374807c70a9551155f089588b6c5e1d10  # [linux64]
+  sha256: bbabb508eb4fb0f9d98ac3ab4a40d0278de41cef95139c1bfe6436b716dd6cfc  # [ppc64le]
+  sha256: 33b3b06cf9838fa3289c7e40f143a8f58b941ccad93f9d0640a29bb83aa2152e  # [aarch64]
+  sha256: b65a34a3715e3ef2b835bdd36c43b21c20800325be1eda1e84ad4d80ee837c92  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+outputs:
+  - name: cuda-cupti
+    files:
+      - lib/libcupti*.so.*                    # [linux]
```

Comment:
Do we not need to ship `libnvperf_host.so` and `libnvperf_target.so` in the `$PREFIX/lib/` folder? If so, please make sure they are tested (these are the files that were missing in the `nvidia` channel package that are needed by nvbench).

Reply:
They are in `-dev` as they are unversioned only.

[Tests below](https://github.com/conda-forge/staged-recipes/pull/22355/files#diff-fdf8b09d413622d288ab00734d612c4ff1cf43048e12cac549314347a6605e9bR89-R90)